docs/guides/getting-started.md

```diff
@@ -16,10 +16,19 @@ npm install --save-dev @humanmade/webpack-helpers
 
 While this package depends in turn on a number of loaders and plugins, it deliberately does _not_ include `webpack` itself. To install this library along with all its relevant peer dependencies, therefore, you may run the following command:
 
+** *Check notice before proceeding:**
 ```bash
 npm install --save-dev @humanmade/webpack-helpers webpack@4 webpack-cli@3 webpack-dev-server sass
 ```
 
+***❗ *Notice***
+This verson is outdated and might leave you with outdated versions of this library and associated Webpack tooling.
+There's a [pending release](https://github.com/humanmade/webpack-helpers/pull/205) that will fix this problem. But for now you should install humanmade/webpack-helpers@beta, webpack @5, webpack-cli@4, and webpack-dev-server@4. To do so runu the following command:
+```bash
+npm install --save-dev @humanmade/webpack-helpers@beta webpack@5 webpack-cli@4 webpack-dev-server@4
+```
+
+
 Note that we specify Webpack version 4. Support for Webpack 5 is anticipated in the v1.0 release of these helpers, but at present using Webpack 4 provides the most predictable and stable experience across our projects.
```

Comment:
May want to move this line up above the notice, since it applies to the old version. Otherwise, this looks great!